### DISCUSSION TITLE
OpenOCR-develop

### DISCUSSION THREAD

**Discussion by Topdu** [Topdu/OpenOCR#33] [General]
OpenOCR det model: openocr_det_repvit_ch.pth (mobile), configs/det/dbnet/repvit_db.yml
OpenOCR rec model: openocr_det_svtrv2_ch.pth (server), configs/rec/svtrv2/svtrv2_ch.yml
OpenOCR rec model: openocr_det_repsvtr_ch.pth (mobile), configs/rec/svtrv2/repsvtr_ch.yml

<hr /><em>This discussion was created from the release <a href='https://github.com/Topdu/OpenOCR/releases/tag/'>OpenOCR-develop</a>.</em>

**Comment by webvokess** [comment]
Good Work ❤️
